File: Arithmetic-Algebra/1051/1051.py
def main():
    num = int(input())
    #print(num)
    count = 0
    if( ( (num-2) /3) == int( (num-2) /3) ):
        count+=1

    #l = '011' * int(num/3)

    print(count + (int(num/3)*2) )

if __name__ == "__main__":
    main()

File: COJ/Arithmetic-Algebra/1051/1051.cpp
//#include <iostream>
#include <bits/stdc++.h> 


using namespace std;


void amount(long long num){
    long long int count = 0;
    if( ( (num*1.0)-2)/3 == int( (num-2)/3 ) )
        count+=1;
    
    cout << count + ( int(num/3)*2 ) << endl; 
}

int main(){
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    long long N;
    cin >> N;
    amount(N);
    return 0;
}
